.clang-tidy

```diff
@@ -0,0 +1,44 @@
+---
+Checks: >
```

Comment:
Why  "-*" is needed?
I thought that plan was enable "performance-faster-string-find", "performance-for-range-copy".
```
Checks: >
  performance-faster-string-find,
  performance-for-range-copy,
```
Are you sure that all "clang-diagnostic-*" , "clang-analyzer-*" checks are already fixed?

Reply:
`clang-tidy` enables a lot of default options. The `-*` option is to remove all options currently enabled.